stingray/crossspectrum.py

```diff
@@ -59,6 +61,13 @@ def __init__(self, lc1=None, lc2=None, norm='none'):
         norm: {'frac', 'abs', 'leahy', 'none'}, default 'none'
             The normalization of the (real part of the) cross spectrum.
 
+        Other Parameters
```

Comment:
`Other Parameters` is usually for parameters that should normally not be touched, but are there if you really know what to do. At least, that's how I interpreted the docs. No strong opinions anyway.